OneCommunityGlobal/HGNRest
# Leo Add ability to create a team from the Add Team option of User Management

# Description
![Screenshot 2023-07-12 001931](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/122568562/df43efea-6405-4641-b02f-5e96eaed2101)

## Related PRS (if any):
To test this backend PR you need to checkout the #997 frontend PR.
…

## Main changes explained:
- Update the postTeam function in teamcontroller.js to avoid user creating duplicate teams.
…

## How to test:
1. check into current branch in backend and frontend
2. do 'npm run build' and 'npm start' to run this PR locally
3. rest, follow frontend PR

## Screenshots or videos of changes:

## Note:
Include the information the reviewers need to know.
